libs/application/templates/general-fishing-license/src/forms/GeneralFishingLicenseForm/fishingLicenseFurtherInfoSection.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  buildCustomField,
+  buildDateField,
+  buildDescriptionField,
+  buildFileUploadField,
+  buildMultiField,
+  buildSection,
+  getValueViaPath,
+} from '@island.is/application/core'
+import { FormValue } from '@island.is/application/types'
+import { fishingLicenseFurtherInformation } from '../../lib/messages'
+import { FishingLicenseEnum } from '../../types'
+import { ATTACHMENTS_FIELD_ID, DATE_FIELD_ID } from '../../utils/fields'
+import {
+  licenseHasAreaSelection,
+  licenseHasFileUploadField,
+  licenseHasRailNetAndRoeNetField,
+  FILE_UPLOAD_ACCEPT,
+} from '../../utils/licenses'
+
+// Condition that determines whether given license has file upload field
+const hasFileUpload = (formValue: FormValue) => {
```

Comment:
Why wrap all the already present util functions?

Reply:
To simplify the ui code below and for better readability - see that I will only have to pass in the function name instead of the entire function itself to the buildFileUploadField condition